### PR TITLE
Adds preflight request functionality and handlers for webcontainer.

### PR DESCRIPTION
**Pull request by awichera**

### CODE REVIEW

File: asab/web/container.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ class WebContainer(ConfigObject):
 		'rootdir': '',
 		'servertokens': 'full',  # Controls whether 'Server' response header field is included ('full') or faked 'prod' ()
 		'cors': '',
-		'preflightpaths':'/openidconnect/*, /test/*',
+		'preflightpaths': '/openidconnect/*, /test/*',
```
Comment (ateska) [inline asab/web/container.py:74]: `cors_preflight_paths`, also allow multiline (see e.g. seacat pki for that bit).

File: asab/log.py [hunk 2]
```diff
@@ -295,7 +295,7 @@ class SyslogRFC3164Formatter(StructuredDataFormatter):
 	def __init__(self, fmt=None, datefmt=None, style='%', sd_id='sd'):
 		fmt = '<%(priority)s>%(asctime)s {app_name} {proc_id} %(levelname)s %(name)s %(struct_data)s%(message)s\000'.format(
 			app_name=Config["logging"]["app_name"],
-			hostname=socket.gethostname(),
```
Comment (ateska) [inline asab/log.py:298]: What is this?

File: asab/web/container.py (hunk 1 above)
Comment (ateska) [reply asab/web/container.py:74]: How did you resolve this request?

File: asab/web/container.py [hunk 3]
```diff
@@ -159,6 +184,10 @@ async def _on_prepare_response(self, request, response):
 		if self.CORS == "*":
 			response.headers['Access-Control-Allow-Origin'] = "*"
 			response.headers['Access-Control-Allow-Methods'] = "GET, POST, DELETE, PUT, PATCH, OPTIONS"
+		elif self.CORS != "":
```
Comment (ateska) [inline asab/web/container.py:187]: This should be just `if self.CORS != ""` (no  elif) ... merge these two together

File: asab/zookeeper/__init__.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
 import logging
 
 from .service import ZooKeeperService
-from .container import ZooKeeperContainer
```
Comment (ateska) [inline asab/zookeeper/__init__.py:4]: Well, you include random irrelevant stuff in merge request ... create a small one for these next time. please.

File: asab/log.py (hunk 2 above)
Comment (awichera) [reply asab/log.py:298]: Well, hostname is not used in the string, flake8 was complaining about it.

File: asab/zookeeper/__init__.py (hunk 4 above)
Comment (awichera) [reply asab/zookeeper/__init__.py:4]: I was trying to make PEP8 happy, to make branch mergable. 
I don't know, why there were these relikts in master. I guess someone have merged something, that didn't pass the unit tests..

File: asab/web/container.py (hunk 1 above)
Comment (awichera) [reply asab/web/container.py]: I allowed multiline and forgot to rename...

**Comment by awichera** [comment]
Travis is failing again on pull request, since there is again some non Pep8 complient stuff in master. I don't have merge conflicts, though.
I don't know what is the correct procedure in that case..

**Pull request merged**